CHANGELOG.md

```diff
@@ -205,7 +205,7 @@ BDK and LDK together.
 - `verify` flag removed from `TransactionDetails`.
 - Add `get_internal_address` to allow you to get internal addresses just as you get external addresses.
 - added `ensure_addresses_cached` to `Wallet` to let offline wallets load and cache addresses in their database
-- Add `is_spent` field to `LocalUtxo`; when we notice that a utxo has been spent we set `is_spent` field to true instead of deleting it from the db.
```

Comment:
I don't think we should be modifying the old changelog - at the time, when we published those versions, LocalOutput was indeed called LocalUtxo :) 

Can you please revert your changes on the CHANGELOG.md file?

Reply:
I got  your point .